src/build/supportedExtensions.js

```diff
@@ -4,7 +4,7 @@ exports.extensions = {
   supported: [
     { icon: 'actionscript', extensions: ['as'] },
     { icon: 'angular', extensions: ['angular-cli.json'], contribType: ctype.filename },
-    { icon: 'apache', extensions: ['htaccess', 'htpasswd'] },
+    { icon: 'apache', extensions: ['htaccess', 'htpasswd'], contribType: ctype.filename },
```

Comment:
@JimiC note that when stating that some extensions is a filename it must be literal. So in this case, you should change `htacccess` to `.htaccess`.

This worked because when it's considered an extension you ommit the initial dot because it's implicit.